server/config/toml.go

```diff
@@ -147,6 +147,28 @@ enable = {{ .GRPC.Enable }}
 # Address defines the gRPC server address to bind to.
 address = "{{ .GRPC.Address }}"
 
+[grpc.grpc-proxy]
+	# GRPCWebProxyEnable defines if the gRPC-web-proxy should be enabled.
+	enable = {{ .GRPC.GRPCWebProxy.Enable }}
+
+	# BindAddress defines address to bind the server to.
+	bind-address = "{{ .GRPC.GRPCWebProxy.BindAddress }}"
+
+	# HTTPPort defines TCP port to listen on for HTTP1.1 debug calls.
+	http-port = {{ .GRPC.GRPCWebProxy.HTTPPort }}
+
+	# AllowAllOrigins defines allow requests from any origin.
+	allow-all-origins = {{ .GRPC.GRPCWebProxy.AllowAllOrigins }}
+
+	# AllowedOrigins defines list of origin URLs which are allowed to make cross-origin requests.
+	allowed-origins = [{{ range $k, $v := .GRPC.GRPCWebProxy.AllowedOrigins }} "{{ $v }}", {{ end }}]
+
+	# AllowedHeaders defines list of headers which are allowed to propagate to the gRPC backend.
+	allowed-headers = [{{ range $k, $v := .GRPC.GRPCWebProxy.AllowedHeaders }} "{{ $v }}", {{ end }}]
+
+	# EnableHTTPServer defines if the HTTP should be enabled.
+	enable-http-server = {{ .GRPC.GRPCWebProxy.EnableHTTPServer }}
```

Comment:
What is the difference between this one and the `enable` field?

Reply:
I have the same question.

Reply:
Yeah, We can remove `enable-http-server` flag. doesn't make sense.